sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ### 4.3.1 (Unreleased)
 
 #### Features Added
-
+- Added the ability to create containers and databases with autoscale properties for the sync and async clients.
```

Comment:
this is just containers since we didn't include the database methods in this PR - either remove database or add the changes to those files as well please